### DISCUSSION TITLE
Forecast returns 503

### DISCUSSION THREAD

**Discussion by swbrains** [weather-gov/api#575] [General]
The following API URL returns a 503 consistently (for more than a week):

https://api.weather.gov/gridpoints/ABQ/124,148/forecast

Going to the NWS website and getting the forecast for the same station ID (KSAF) works fine.

Using the old mapClick.php XML link on the website page works as well and returns valid forecast data:
https://forecast.weather.gov/MapClick.php?lat=35.7094&lon=-106.0555&unit=0&lg=english&FcstType=dwml

Our code uses the API to get the weather conditions and forecast for any US ZIP code and seems to work properly for other ZIP codes we've tested with.  It seems to be an issue with only this location, but it has persisted for quite a while.

Here is the HTTP return content from the API:
Unavailable
Cache-Control: private, must-revalidate, max-age=3561
Connection: close
Date: Fri, 12 Aug 2022 00:02:36 GMT
Pragma: no-cache
Server: nginx/1.20.1
Vary: Accept,Feature-Flags,Accept-Language
Content-Length: 311
Content-Type: application/problem+json
Expires: Fri, 12 Aug 2022 01:01:57 GMT
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: X-Correlation-Id, X-Request-Id, X-Server-Id
Client-Date: Fri, 12 Aug 2022 00:02:36 GMT
Client-Peer: 23.200.224.17:443
Client-Response-Num: 1
Client-SSL-Cert-Issuer: /C=US/O=DigiCert Inc/CN=DigiCert TLS RSA SHA256 2020 CA1
Client-SSL-Cert-Subject: /C=US/ST=Maryland/L=College Park/O=National Oceanic and Atmospheric Administration/CN=weather.gov
Client-SSL-Cipher: ECDHE-RSA-AES256-GCM-SHA384
Client-SSL-Socket-Class: IO::Socket::SSL
Client-SSL-Warning: Peer certificate not verified
Strict-Transport-Security: max-age=31536000 ; includeSubDomains ; preload
X-Correlation-ID: 36e1c798
X-Edge-Request-ID: 99288ac
X-Request-ID: e1ea9180-eead-496b-8e52-231ce1e3aa3c
X-Server-ID: vm-bldr-nids-apiapp6.ncep.noaa.gov

{
    "correlationId": "36e1c798",
    "title": "Service Unavailable",
    "type": "https://api.weather.gov/problems/ServiceUnavailable",
    "status": 503,
    "detail": "An upstream data source is temporarily unavailable. Please try again later.",
    "instance": "https://api.weather.gov/requests/36e1c798"
}

Since the web site can get data for this location, I'm wondering what we need to do to get the same forecast data through the current API interface. 

Thanks!

**Comment by JJKraw** [comment]
Have you sent this to nco.ops@noaa.gov ?

Looks like an operational issue.

**Comment by scadergit** [comment]
That is a good idea. They will tell you it is an existing issue, but always
good to remind them. I don't have the full details, but it's something to
do with the way the Field office create the netcdf. Most offices have fixed
it, but there are two outliers including ABQ.

**Comment by swbrains** [comment]
Thanks!
